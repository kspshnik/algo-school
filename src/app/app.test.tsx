import React from 'react';
import { render } from '@testing-library/react';
import App from './app';

describe('App should not change main page during development', () => {
  const app = render(<App />);
  it('should match snapshot', () => {
    expect(app).toMatchSnapshot();
  });
});
