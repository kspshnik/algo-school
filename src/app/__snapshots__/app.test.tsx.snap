// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should not change main page during development should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="app"
      >
        <main
          class="content "
        >
          <div
            class="title_box"
          >
            <h1
              class="text text_type_h1 text_color_h1 title"
            >
              МБОУ АЛГОСОШ
            </h1>
            <p
              class="text text_type_fibonacci text_color_secondary fibonacci_title"
            >
              им. Фибоначчи
            </p>
          </div>
          <div
            class="cards_box"
          >
            <a
              class="link"
              href="/recursion"
            >
              <div
                class="card string"
              />
            </a>
            <a
              class="link"
              href="/fibonacci"
            >
              <div
                class="card fibonacci"
              />
            </a>
            <a
              class="link"
              href="/sorting"
            >
              <div
                class="card arr"
              />
            </a>
            <a
              class="link"
              href="/stack"
            >
              <div
                class="card stack"
              />
            </a>
            <a
              class="link"
              href="/queue"
            >
              <div
                class="card queue"
              />
            </a>
            <a
              class="link"
              href="/list"
            >
              <div
                class="card list"
              />
            </a>
          </div>
          <div
            class="ticker marquee-container"
            style="--pause-on-hover: running; --pause-on-click: running;"
          >
            <div
              class="marquee"
              style="--play: running; --direction: normal; --duration: 0s; --delay: 0s; --iteration-count: infinite;"
            >
              <p
                class="text text_type_ticker text_color_secondary ticker_text"
              >
                Вдохновлено школами, в которых не учили алгоритмам
              </p>
              <div
                class="dot_box"
              >
                <p
                  class="dot"
                />
              </div>
            </div>
            <div
              class="marquee"
              style="--play: running; --direction: normal; --duration: 0s; --delay: 0s; --iteration-count: infinite;"
            >
              <p
                class="text text_type_ticker text_color_secondary ticker_text"
              >
                Вдохновлено школами, в которых не учили алгоритмам
              </p>
              <div
                class="dot_box"
              >
                <p
                  class="dot"
                />
              </div>
            </div>
          </div>
          <p
            class="text text_type_column text_color_input mt-14 copyright"
          >
            © Сделано в Практикуме.
          </p>
        </main>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="app"
    >
      <main
        class="content "
      >
        <div
          class="title_box"
        >
          <h1
            class="text text_type_h1 text_color_h1 title"
          >
            МБОУ АЛГОСОШ
          </h1>
          <p
            class="text text_type_fibonacci text_color_secondary fibonacci_title"
          >
            им. Фибоначчи
          </p>
        </div>
        <div
          class="cards_box"
        >
          <a
            class="link"
            href="/recursion"
          >
            <div
              class="card string"
            />
          </a>
          <a
            class="link"
            href="/fibonacci"
          >
            <div
              class="card fibonacci"
            />
          </a>
          <a
            class="link"
            href="/sorting"
          >
            <div
              class="card arr"
            />
          </a>
          <a
            class="link"
            href="/stack"
          >
            <div
              class="card stack"
            />
          </a>
          <a
            class="link"
            href="/queue"
          >
            <div
              class="card queue"
            />
          </a>
          <a
            class="link"
            href="/list"
          >
            <div
              class="card list"
            />
          </a>
        </div>
        <div
          class="ticker marquee-container"
          style="--pause-on-hover: running; --pause-on-click: running;"
        >
          <div
            class="marquee"
            style="--play: running; --direction: normal; --duration: 0s; --delay: 0s; --iteration-count: infinite;"
          >
            <p
              class="text text_type_ticker text_color_secondary ticker_text"
            >
              Вдохновлено школами, в которых не учили алгоритмам
            </p>
            <div
              class="dot_box"
            >
              <p
                class="dot"
              />
            </div>
          </div>
          <div
            class="marquee"
            style="--play: running; --direction: normal; --duration: 0s; --delay: 0s; --iteration-count: infinite;"
          >
            <p
              class="text text_type_ticker text_color_secondary ticker_text"
            >
              Вдохновлено школами, в которых не учили алгоритмам
            </p>
            <div
              class="dot_box"
            >
              <p
                class="dot"
              />
            </div>
          </div>
        </div>
        <p
          class="text text_type_column text_color_input mt-14 copyright"
        >
          © Сделано в Практикуме.
        </p>
      </main>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
